EJ2-Documentation(846715) - UG documentation for the Row-template

Gayathri4135 opened the pull request:
Modified the rowtemplate in row for js and ts

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/170/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/170/)<br/>**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/203/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/203/)<br/>**

Gayathri4135 commented:
Changes reviewed in this PR: https://github.com/syncfusion-content/ej2-grid-docs/pull/36
The reviewed changes :
	1. Remove the unnecessary properties.
	Removed the unnecessary properties.
	2. Change the content of limitation.
	Changed the content of limitation.
	3. Update the version in the sample.
        Updated the version in the sample

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/282/console)**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/282/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-rowTemplate-846715](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/282/)<br/>**

BalajiSGrid commented:
The demo and technical review have been completed.

MaithiliyK commented:
this PR is prepared in old hotfix branch moved the PR to new branch https://github.com/syncfusion-content/ej2-grid-docs/pull/130/files. So i closed this task
